README.md

```diff
@@ -1,12 +1,60 @@
 # [qri](http://qri.io)
 
-[![Qri](https://img.shields.io/badge/made%20by-qri-magenta.svg?style=flat-square)](https://qri.io)
-[![GoDoc](https://godoc.org/github.com/qri-io/qri?status.svg)](http://godoc.org/github.com/qri-io/qri)
-[![License](https://img.shields.io/github/license/qri-io/qri.svg?style=flat-square)](./LICENSE)
-[![Codecov](https://img.shields.io/codecov/c/github/qri-io/qri.svg?style=flat-square)](https://codecov.io/gh/qri-io/qri)
-[![CI](https://img.shields.io/circleci/project/github/qri-io/qri.svg?style=flat-square)](https://circleci.com/gh/qri-io/qri)
-
-#### qri is a global dataset version control system (GDVCS) built on the distributed web
+[![Qri](https://img.shields.io/badge/made%20by-qri-magenta.svg?style=flat-square)](https://qri.io) [![GoDoc](https://godoc.org/github.com/qri-io/qri?status.svg)](http://godoc.org/github.com/qri-io/qri) [![License](https://img.shields.io/github/license/qri-io/qri.svg?style=flat-square)](./LICENSE) [![Codecov](https://img.shields.io/codecov/c/github/qri-io/qri.svg?style=flat-square)](https://codecov.io/gh/qri-io/qri) [![CI](https://img.shields.io/circleci/project/github/qri-io/qri.svg?style=flat-square)](https://circleci.com/gh/qri-io/qri)
+
+<h1 align="center">Qri Backend and CLI</h1>
+
+<div align="center">
+  <img alt="logo" src="https://qri.io/img/blobs/blob_trio.png" width="128">
```

Comment:
I think the backend should get it's own set of blobs to help visually distinguish frontend & backend. This is an issue  we should all clearly discuss as a team. So let's merge this and make more blobs.